ports/irsdkcpp/portfile.cmake

```
@@ -0,0 +1,30 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO vrkit-platform/irsdk-cpp
+    REF "v${VERSION}"
+    SHA512 efdaafc2badcce7ba88fb943882aa301fa3c9959c8de86a99269ade05f188828c485e16f21241d558b108d48e566702e961a1c5b0bda42e523abcc783bdde5c6
+    HEAD_REF develop

```

```suggestion
    HEAD_REF master
```
The default branch of [upsteam](https://github.com/[vrkit-platform/irsdk-cpp](https://github.com/vrkit-platform/irsdk-cpp)) is master

---

We merge onto the develop branch, only releases hit main. Therefore in our library, `develop` is upstream (or next release/head).